### ISSUE TITLE
Error about swift binary not available, even when not using swift

### ISSUE THREAD

**Issue by Tyilo** [theos/theos#294]
What are the steps to reproduce this issue?
-------------------------------------------
1. Create a new theos project:
```
$ nic.pl
NIC 2.0 - New Instance Creator
------------------------------
  [1.] iphone/activator_event
  [2.] iphone/application_modern
  [3.] iphone/cydget
  [4.] iphone/flipswitch_switch
  [5.] iphone/framework
  [6.] iphone/ios7_notification_center_widget
  [7.] iphone/library
  [8.] iphone/notification_center_widget
  [9.] iphone/preference_bundle_modern
  [10.] iphone/tool
  [11.] iphone/tweak
  [12.] iphone/xpc_service
Choose a Template (required): 10
Project Name (required): foobar
Package Name [com.yourcompany.foobar]: 
Author/Maintainer Name [tyilo]: 
Instantiating iphone/tool in foobar/...
Done.
```
2. Run make:
```
$ make
bash: /home/tyilo/theos/toolchain/linux/swift/bin/swift: No such file or directory
> Making all for tool foobar…
bash: /home/tyilo/theos/toolchain/linux/swift/bin/swift: No such file or directory
bash: /home/tyilo/theos/toolchain/linux/swift/bin/swift: No such file or directory
bash: /home/tyilo/theos/toolchain/linux/swift/bin/swift: No such file or directory
bash: /home/tyilo/theos/toolchain/linux/swift/bin/swift: No such file or directory
==> Compiling main.mm (armv7)…
==> Compiling main.mm (arm64)…
==> Linking tool foobar (armv7)…
==> Linking tool foobar (arm64)…
==> Merging tool foobar…
==> Signing foobar…
```

What happens?
-------------
I get an error, because I don't have a swift toolchain installed even though I'm only compiling Objective-C.

What were you expecting to happen?
----------------------------------
No error.

Any logs, error output, etc?
----------------------------
See above.

Any other comments?
-------------------

What versions of software are you using?
----------------------------------------
**Operating System:** Linux

**Toolchain Version:**

```
iOS Toolchain for Linux

LLVM + Clang 3.9.1
LD64 264.3.102
CCTools 877.5

Built by CoolStar w/ <3
```

**SDK Version:** `iPhoneOS9.3.sdk`

**Comment by kirb** [comment]
Fixed in the [clean-up-targets](https://github.com/theos/theos/tree/kirb/implement/clean-up-targets) branch that will be merged soon. It's a harmless error, it's just us calling `swift --version` to get the version of Swift installed.

**Comment by kirb** [comment]
Branch was merged in 2.4.
